### PR TITLE
updating final form and final-form-arrays version #169613890

### PR DESCRIPTION
**Pull request by LoganArnett**
This PR is for ticket [#169613890](https://www.pivotaltracker.com/story/show/169613890)

### CODE REVIEW

**Comment by akegan** [comment]
That's a lot of snapshot changes, do you have a summary of what we expect them to be from?

File: spec/javascript/components/applications/__snapshots__/ApplicationEditPage.test.js.snap [hunk 1]
```diff
@@ -1614,7 +1614,6 @@ Array [
                                 onChange={[Function]}
                                 onFocus={[Function]}
                                 type="checkbox"
-                                value={true}
```
Comment (LoganArnett) [inline spec/javascript/components/applications/__snapshots__/ApplicationEditPage.test.js.snap:1617]: These values are removed from the snapshots due to default values being assumed by the Final Form fields

File: spec/javascript/components/applications/__snapshots__/ApplicationNewPage.test.js.snap [hunk 2]
```diff
@@ -1194,7 +1194,6 @@ Array [
                                 onChange={[Function]}
                                 onFocus={[Function]}
                                 type="checkbox"
-                                value=""
```
Comment (LoganArnett) [inline spec/javascript/components/applications/__snapshots__/ApplicationNewPage.test.js.snap:1197]: These values are removed from the snapshots due to default values being assumed by the Final Form fields

File: spec/javascript/components/lease_ups/__snapshots__/LeaseUpApplicationsPage.test.js.snap [hunk 3]
```diff
@@ -314,94 +314,66 @@ exports[`LeaseUpApplicationsPage Should render LeaseUpTable 1`] = `
                               }
                               placeholder="Preference"
                             >
-                              <ReactFinalForm(Field)
```
Comment (LoganArnett) [inline spec/javascript/components/lease_ups/__snapshots__/LeaseUpApplicationsPage.test.js.snap:317]: These wrappers are from the updated API resolving to the Field directly

File: spec/javascript/components/applications/__snapshots__/ApplicationEditPage.test.js.snap [hunk 4]
```diff
@@ -1697,89 +1694,6 @@ Array [
                 <h3>
                   Preferences
                 </h3>
-                <div
```
Comment (akegan) [inline spec/javascript/components/applications/__snapshots__/ApplicationEditPage.test.js.snap:1700]: why do you think this preference section went away?

**Comment by akegan** [comment]
OK, I tested:

**Paper App**
- Checking validation by trying to submit an empty form
- Submitting an application with just the required fields
- Submitting an application with all of the fields
- Testing adding and removing HH members and prefs
- Testing validation around alt contacts, HH members, and prefs


**Supp app**
- Updating fields
- updating confirmed prefs and having lease update

**Review by akegan — APPROVED** [review]
Looks good overall, just have that one last question on the updated snapshot then it's good to merge

**Pull request merged**